src/Symfony/Component/Debug/Tests/DebugClassLoaderTest.php

```diff
@@ -133,7 +133,8 @@ class_exists(__NAMESPACE__.'\TestingCaseMismatch', true);
      */
     public function testFileCaseMismatch()
     {
-        if (!file_exists(__DIR__.'/Fixtures/CaseMismatch.php')) {
+        //Cannot be tested if the file loads (there is is casemismatch.php in the filesystem)
+        if (file_exists(__DIR__.'/Fixtures/CaseMismatch.php')) {
```

Comment:
Sure, I just wanted to agree that the test looks OK before I start investigating further.

Or maybe someone else with Linux/Mac will? (Because this would be hard to debug on Windows)